app/root.tsx

```diff
@@ -133,12 +133,26 @@ function App() {
 	const theme = useTheme()
 
 	return (
-		<html lang="en" className={`${theme} h-full`}>
+		<html
+			lang="en"
+			className={`${theme} h-full`}
+			style={{ colorScheme: theme }}
+		>
 			<head>
 				<ClientHintCheck nonce={nonce} />
 				<Meta />
 				<meta charSet="utf-8" />
 				<meta name="viewport" content="width=device-width,initial-scale=1" />
+				<meta
+					name="theme-color"
+					media="(prefers-color-scheme: light)"
+					content="white"
```

Comment:
I think we'll want these to be one of our tailwind colors. Unfortunately the value of this attribute cannot be a CSS variable so we'll either need to read it from the tailwind config at runtime in the loader and pass it into here or we'll have to hard-code it. I'm not sure which of those options I dislike least. Let's remove the theme-color for now until our design settles a bit.

Reply:
I looked into this yesterday and was going to get back to you with the same response. Content attribute cannot be a variable. 

I believe a solution maybe to use CSS media query. And then I cam e to the same conclusion that your css theme isn't quite there yet to super this. This is conflating anyway with the feature of this PR (`color-scheme`) so I have removed `theme-colour` support.

```
		@media (prefers-color-scheme: light) {
			meta[name="theme-color"] {
				content: var(--background);
			}
		}

		@media (prefers-color-scheme: dark) {
			meta[name="theme-color"] {
				content: var(--background);
			}
		}
```